api/src/main/java/io/opentelemetry/baggage/propagation/ContextKeys.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.baggage.propagation;
+
+import io.opentelemetry.context.Context;
+
+public final class ContextKeys {
+  private static final Context.Key<Object> BAGGAGE_KEY = Context.createKey("baggage");
+
+  public static Context.Key<Object> getSpanContextKey() {
+    return BAGGAGE_KEY;
```

Comment:
Should there be a separate key for span context or is actually being stored in baggage?

Reply:
Separate key for `SpanContext` ;)

Reply:
I see, there is a `getSpanContextKey` for each context (key).